includes/admin.form.inc

```diff
@@ -351,22 +353,1290 @@ function islandora_scholar_admin_pdf_form($form, &$form_state) {
+  form_load_include($form_state, 'inc', 'islandora_scholar', 'includes/admin.form');
+  if (!empty($form_state['values']['islandora_scholar_xpaths_demo'])) {
+    $islandora_scholar_xpath_pid = $form_state['values']['islandora_scholar_xpaths_demo'];
+    exec($islandora_scholar_xpath_pid, $output, $return_value);
+    $islandora_scholar_xpaths_general_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field. <span style="color:blue">Testing against pid:</span> <span style="color:green">!pid</span> <img src="', array(
+        '!pid' => "$islandora_scholar_xpath_pid",
+      )) . url('misc/watchdog-ok.png') . '"/><br/><br/>';
+    $islandora_scholar_xpaths_google_description = t('Not limited to Google Scholar, most of these are used by Coins and Citeproc as well. To see the MODS value after editing an entry, tab to the next field. <span style="color:blue">Testing against pid:</span> <span style="color:green">!pid</span> <img src="', array(
+        '!pid' => "$islandora_scholar_xpath_pid",
+      )) . url('misc/watchdog-ok.png') . '"/><br/><br/>';
+    $islandora_scholar_xpaths_coins_description = t('XPaths for the construction of COINS URLs. To see the MODS value after editing an entry, tab to the next field. <span style="color:blue">Testing against pid:</span> <span style="color:green">!pid</span> <img src="', array(
+        '!pid' => "$islandora_scholar_xpath_pid",
+      )) . url('misc/watchdog-ok.png') . '"/><br/><br/>';
+    $islandora_scholar_xpaths_citeproc_description = t('MODS to Citeproc JSON Submodule Converter Xpath configurations <span style="color:blue">Testing against pid:</span> <span style="color:green">!pid</span> <img src="', array(
+        '!pid' => "$islandora_scholar_xpath_pid",
+      )) . url('misc/watchdog-ok.png') . '"/><br/><br/>';
+  }
+  else {
+    $islandora_scholar_xpaths_general_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field.');
+    $islandora_scholar_xpaths_google_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field.');
+    $islandora_scholar_xpaths_coins_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field.');
+    $islandora_scholar_xpaths_citeproc_description = t('These XPaths are shared within most of the submodules. To see the MODS value after editing an entry, tab to the next field.');
+  }
+  $form = array(
+    'islandora_scholar_xpaths' => array(
+      '#type' => 'fieldset',
+      '#title' => t('Islandora Scholar Xpath configuration'),
+      '#description' => t("Set custom XPaths for Islandora Scholar. <br/><br/>
+      Additional XPath Configurations !gs_search_xpaths_configuration & !library_catalog_search_xpaths_configuration<br/><br/>",
```

Comment:
I'm still not a fan of this line existing on the config page. I think this info belongs in the README only.

Reply:
I hate GitHub's reviews UI, so in case you can't tell what I'm pointing to, its the part of the admin page UI that says `Additional XPath Configurations Google Scholar Search XPaths & Library Catalog Search XPaths Configuration Page`.

IMHO thats taking up space on an already busy page with information that rightly belongs in the README or JIRA documentation.

Reply:
OK, no problem. I can make that move. I left it in because of the name "XPaths" implies all XPaths are located on that tab but there are a few elsewhere.